src/FSharpLint.Core/fsharplint.json

```diff
@@ -201,11 +201,18 @@
             "naming": "PascalCase",
             "underscores": "None"
         }
+    },
+	  "genericTypesNames": {
+        "enabled": true,
+        "config": {
+            "naming": "PascalCase",  
+            "underscores": "None"
+        }
     },
     "publicValuesNames": {
         "enabled": true,
         "config": {
-            "underscores": "AllowPrefix"
```

Comment:
@su8898 this change here is unrelated, can you drop it from the PR please?

Reply:
@knocte I've removed the misalignment and updated the PR. Could you please have a look again? thank you.